pyproject.toml

```diff
@@ -1,13 +1,40 @@
+# Testing tools configuration
 [tool.coverage.run]
 branch = true
 
 [tool.coverage.report]
 show_missing = true
 
+[tool.pytest.ini_options]
+minversion = "6.0"
+log_cli_level = "INFO"
+
+# Formatting tools configuration
 [tool.black]
 line-length = 99
 target-version = ["py38"]
 
+# Linting tools configuration
+[tool.ruff]
```

Comment:
We're adding ruff related validation but we're not using ruff in `tox`. Make sure we completely make the switch to ruff here.

Reply:
Done, please note that `ruff` replaces `isort` added in PR description too.